shared/xccdf/system/software/integrity.xml

```diff
@@ -98,23 +98,153 @@ must be captured and it should be able to be verified against the installed file
+negative consequences that could ultimately affect the security state of the operating
+system. The operating system's Information Management Officer (IMO)/Information System
+Security Officer (ISSO) and System Administrators (SAs) must be notified via email and/or
+monitoring system trap when there is an unauthorized modification of a configuration item.
+</rationale>
+<ident prodtype="rhel7" cce="CCE-TBD" />
+<oval id="aide_scan_notification"/>
+<ref prodtype="rhel7" nist="CM-3(5)" disa="1744"
+ossrg="SRG-OS-000363-GPOS-00150" sitgid="020140" />
+</Rule>
+
+<Rule id="aide_verify_acls" severity="medium" prodtype="rhel7">
+<title>Configure AIDE to Verify Access Control Lists (ACLs)</title>
+<description>
+By default, the <tt>acl</tt> option is added to the <tt>FIPSR</tt> ruleset in AIDE.
+If using a custom ruleset or the <tt>acl</tt> option is missing, add <tt>acl</tt>
+to the appropriate ruleset.
+For example, add <tt>acl</tt> to the following line in <tt>/etc/aide.conf</tt>:
+<pre>FIPSR = p+i+n+u+g+s+m+c+acl+selinux+xattrs+sha256</pre>
+AIDE rules can be configured in multiple ways; this is merely one example that is already
+configured by default.
+</description>
+<ocil clause="the acl option is missing or not added to the correct ruleset">
+To determine that AIDE is verifying ACLs, run the following command:
+<pre>$ grep acl /etc/aide.conf</pre>
+Verify that the <tt>acl</tt> option is added to the correct ruleset.
+</ocil>
+<rationale>
+ACLs can provide permissions beyond those permitted through the file mode and must be
+verified by the file integrity tools.
+</rationale>
+<ident prodtype="rhel7" cce="CCE-TBD" />
+<oval id="aide_verify_acls"/>
+<ref prodtype="rhel7" nist="SI-7.1" disa="366"
+ossrg="SRG-OS-000480-GPOS-00227" sitgid="021600" />
+</Rule>
+
+<Rule id="aide_verify_ext_attributes" severity="medium" prodtype="rhel7">
+<title>Configure AIDE to Verify Extended Attributes</title>
+<description>
+By default, the <tt>xattrs</tt> option is added to the <tt>FIPSR</tt> ruleset in AIDE.
+If using a custom ruleset or the <tt>xattrs</tt> option is missing, add <tt>xattrs</tt>
+to the appropriate ruleset.
+For example, add <tt>xattrs</tt> to the following line in <tt>/etc/aide.conf</tt>:
+<pre>FIPSR = p+i+n+u+g+s+m+c+acl+selinux+xattrs+sha256</pre>
+AIDE rules can be configured in multiple ways; this is merely one example that is already
+configured by default.
+</description>
+<ocil clause="the xattrs option is missing or not added to the correct ruleset">
+To determine that AIDE is verifying extended file attributes, run the following command:
+<pre>$ grep xattrs /etc/aide.conf</pre>
+Verify that the <tt>xattrs</tt> option is added to the correct ruleset.
+</ocil>
+<rationale>
+Extended attributes in file systems are used to contain arbitrary data and file metadata
+with security implications.
+</rationale>
+<ident prodtype="rhel7" cce="CCE-TBD" />
+<oval id="aide_verify_ext_attributes"/>
```

Comment:
Opened https://github.com/OpenSCAP/scap-security-guide/issues/1564 to track this needs OVAL